test/integration/integration_admin_test.cc

```diff
@@ -449,7 +449,7 @@ TEST_F(IntegrationAdminIpv4Ipv6Test, Ipv4Ipv6Listen) {
 class StatsMatcherIntegrationTest : public HttpIntegrationTest, public testing::Test {
 public:
   StatsMatcherIntegrationTest()
-      : HttpIntegrationTest(Http::CodecClient::Type::HTTP1, Network::Address::IpVersion::v4,
+      : HttpIntegrationTest(Http::CodecClient::Type::HTTP1, Network::Address::IpVersion::v6,
```

Comment:
Do you think it would be good to use `getIpVersionsForTest()`? That will enable running on environments that don't have v4, or v6.